aptihramy-frontend/src/pages/LoginPage.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <v-col class="fill-height" fluid>
+        <v-row align="center" justify="center" class="mb-4">
+            <v-col cols="12" class="text-center">
+                <h1 class="welcome-title">Welcome Back 👋</h1>
+                <p class="subtitle">Please login to continue</p>
+            </v-col>
+        </v-row>
+
+        <v-row align="center" justify="center">
+            <v-col cols="12" sm="8" md="4">
+                <v-card class="pa-6" elevation="10">
+                    <v-card-title class="text-center text-h5 font-weight-bold">Login</v-card-title>
+
+                    <v-card-text>
+                        <v-form @submit.prevent="handleLogin" ref="formRef" v-model="formValid">
+                            <v-text-field v-model="email" :rules="[requiredRule, emailRule]" label="Email" required
+                                prepend-inner-icon="mdi-account"></v-text-field>
+
+                            <v-text-field v-model="password" :rules="[requiredRule]" label="Password" type="password"
+                                required prepend-inner-icon="mdi-lock"></v-text-field>
+
+                            <v-btn type="submit" color="primary" block class="mt-4">
+                                Login
+                            </v-btn>
+                        </v-form>
+                    </v-card-text>
+                </v-card>
+            </v-col>
+        </v-row>
+    </v-col>
+</template>
+
+
+
+<script setup lang="ts">
+import { ref, computed } from 'vue'
+import { useRouter } from 'vue-router'
+import { login } from '@/core/api'
+import { useErrorMessagesStore } from '@/core/stores/errorMessages';
+import '../styles/main.css';
+
+
+const errorMessagestore = useErrorMessagesStore()
+const queier = computed(() => errorMessagestore.getQueue)
+
+
+const email = ref('')
+const password = ref('')
+const router = useRouter()
+const formRef = ref()
+const formValid = ref(true)
+
+// Rules
+const requiredRule = (v: string) => !!v || 'This field is required'
+const emailRule = (v: string) =>
+    /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(v) || 'Must be a valid email'
+
+
+async function handleLogin() {
+    const isValid = await formRef.value?.validate()
+    if (!isValid.valid) return
+
+    login(email.value, password.value)
+        .then((token) => {
+            localStorage.setItem('token', token)
```

Comment:
Here you should use the `setToken` function, or you could also set the token directly in the `login` function.